superset-frontend/src/explore/exploreUtils/getParsedExploreURLParams.test.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { getParsedExploreURLParams } from './getParsedExploreURLParams';
+
+const setupLocation = (newUrl: string) => {
+  delete (window as any).location;
+  // @ts-ignore
+  window.location = new URL(newUrl);
+};
+
+test('get form_data_key and slice_id from search params - url when moving from dashboard to explore', () => {
+  setupLocation(
+    'http://localhost:9000/superset/explore/?form_data_key=yrLXmyE9fmhQ11lM1KgaD1PoPSBpuLZIJfqdyIdw9GoBwhPFRZHeIgeFiNZljbpd&slice_id=56',
```

Comment:
Maybe extract `http://localhost:9000/superset` to a const and reuse it in the tests?